### ISSUE TITLE
Translate to Spanish metainfo for the Endless Key flatpak

### ISSUE THREAD

**Issue by jofilizola** [endlessm/endless-key-flatpak#21]
Once the metainfo is adapted, provide translations: https://github.com/endlessm/endless-key-flatpak/issues/20

**Comment by jprvita** [comment]
I believe this is a duplicate of #6.

**Comment by jofilizola** [comment]
@jprvita I'm reopening as this is not a duplicate, it's a task for me to translate the text and provide translations so #6 can happen. Makes sense?

**Comment by jprvita** [comment]
> @jprvita I'm reopening as this is not a duplicate, it's a task for me to translate the text and provide translations so #6 can happen. Makes sense?

Yes, sorry about the confusion.

**Comment by jofilizola** [comment]
https://docs.google.com/spreadsheets/d/1yVTilb6AWsyUGVncyoYSjtbOdPAEZA_BarnnTfUDxfg/edit#gid=1177927074

**Comment by jofilizola** [comment]
This is for the actual translated text, Ive just prepared the document for Rodrigo to translate

**Comment by manuq** [comment]
@jofilizola I have reviewed the document already https://docs.google.com/spreadsheets/d/1yVTilb6AWsyUGVncyoYSjtbOdPAEZA_BarnnTfUDxfg/edit#gid=1177927074

**Comment by jofilizola** [comment]
Thanks!
Translated screeshots here: https://www.dropbox.com/scl/fo/nxqdtyyvpkatbl0gcgqcq/h?rlkey=l524pdllrbxkkomm89qmw2ujf&dl=0
Translated copy here: https://docs.google.com/spreadsheets/d/1yVTilb6AWsyUGVncyoYSjtbOdPAEZA_BarnnTfUDxfg/edit#gid=1177927074

**Comment by jofilizola** [comment]
I created a ticket for applying the translations to the actual listings https://github.com/endlessm/kolibri-explore-plugin/issues/855

Can we close this?

**Comment by manuq** [comment]
> I created a ticket for applying the translations to the actual listings [endlessm/kolibri-explore-plugin#855](https://github.com/endlessm/kolibri-explore-plugin/issues/855)
> 
> Can we close this?

Yep. Although I don't think kolibri-explore-plugin is the right target for applying the translations. Metainfo is in this one (endless-key-flatpak).

**Comment by vanessa-chang** [comment]
It's verified passed in the Chrome playstore.
